pages/api/webhooks/customer-created.ts

```diff
@@ -0,0 +1,43 @@
+import { NextApiRequest, NextApiResponse, NextApiHandler } from "next";
+import { SALEOR_DOMAIN_HEADER } from "../../../constants";
+import Klaviyo from "../../../lib/klaviyo";
+
+const expectedEvent = "customer_created";
+
+const handler: NextApiHandler = async (
```

Comment:
if `handler: NextApiHandler`, there's no reason to define `request: NextApiRequest` and `response: NextApiResponse` - they are inferred by TypeScript from `handler` type.

Nothing wrong with having them explicit, it will just break standard, that we have established.

Reply:
Yes, exactly that was my initial intention. Let's relay on the **type inference** as much as possible @Cloud11PL